commit 3036b2933fde
Author: Gijs Kruitbosch <gijskruitbosch@gmail.com>
Date:   Tue Oct 3 19:29:36 2017 +0100

    Bug 1405377 - Remove trailing separator in recently closed tabs/windows subviews. r=mikedeboer, a=ritu
    
    The third param of 'getTabsFragment' and 'getWindowsFragment' toggles whether the 'restore all' item
    gets prefixed (true) or suffixed (false). The prefixed version doesn't get a separator, so this
    seems like the simplest fix.
    
    MozReview-Commit-ID: BzKWvndWUMp
    
    --HG--
    extra : source : ed7b92c9eca5e1faa1b193918f0b9ddf7fc9c0e5
---
 browser/components/customizableui/CustomizableWidgets.jsm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git browser/components/customizableui/CustomizableWidgets.jsm browser/components/customizableui/CustomizableWidgets.jsm
index fb12d40da42d..b197d2d72fdc 100644
--- browser/components/customizableui/CustomizableWidgets.jsm
+++ browser/components/customizableui/CustomizableWidgets.jsm
@@ -215,7 +215,7 @@ const CustomizableWidgets = [
 
       let utils = RecentlyClosedTabsAndWindowsMenuUtils;
       let method = `get${viewType}Fragment`;
-      let fragment = utils[method](window, "toolbarbutton");
+      let fragment = utils[method](window, "toolbarbutton", true);
       let elementCount = fragment.childElementCount;
       this._panelMenuView._setEmptyPopupStatus(panelview, !elementCount);
       if (!elementCount)
